packages/config/src/android/Colors.ts

```diff
@@ -56,3 +56,14 @@ export function setColorItem(itemToAdd: XMLItem[], colorFileContentsJSON: Docume
   }
   return colorFileContentsJSON;
 }
+
+export function removeColorItem(keyToRemove: string, colorFileContentsJSON: Document) {
```

Comment:
used?

Reply:
nope not used, we default to setting the background color to white so we don't need this, I'll take it out